### PR TITLE
feat: 🎸 Add gcloud route to application

### PR DESCRIPTION
**Pull request by ladeirarodolfo**
Added gcloud full route to select store/location/project/study/series
### Includes
- Viewer to load url based on gcloud API url pattern
- Uses static gcloud API url pattern
- Persist servers/gcloud info on session storage

### A step to
- issue #879 
- Generic route handler (based on app config)

### Screenshoot
Oct-10-2019-2_38-PM

Closes: #753

### CODE REVIEW

**Comment by swederik** [comment]
In case anyone else is trying to test this, and has signed the TTA for access to the TCIA collections on Google Cloud Healthcare:

1. Checkout the branch
2. Run the viewer with your client ID in the config file 
3. Navigate to a URL like this:
http://localhost:3000/projects/chc-tcia/locations/us-central1/datasets/qin-headneck/dicomStores/qin-headneck/dicomWeb/study/1.3.6.1.4.1.14519.5.2.1.2744.7002.292382393135745002321454178686

File: platform/viewer/src/googleCloud/api/GoogleCloudApi.js [hunk 1]
```diff
@@ -22,6 +22,13 @@ class GoogleCloudApi {
     return this.urlBase + `/projects`;
   }
 
+  getUrlBaseDicomWeb(project, location, dataset, dicomstore) {
+    return (
+      this.urlBase +
+      `/projects/${project}/locations/${location}/datasets/${dataset}/dicomStores/${dicomstore}/dicomWeb`
```
Comment (swederik) [inline platform/viewer/src/googleCloud/api/GoogleCloudApi.js]: For me this PR is good, but I would remove the`/dicomWeb` at the end. Thoughts @pieper?

Comment (pieper) [reply platform/viewer/src/googleCloud/api/GoogleCloudApi.js]: I agree, no need for `dicomWeb`.

Otherwise yes, LGTM 👍

Comment (ladeirarodolfo) [reply platform/viewer/src/googleCloud/api/GoogleCloudApi.js]: Ok removed it..

**Comment by ladeirarodolfo** [comment]
Updated PR with more improvements: 
Store gcloud info on session storage (reduced requirement to set gcloud data everytime)
Check it out screenshoot

**Comment by swederik** [comment]
Latest changes are working well for me, and the code changes look good. This makes Google Cloud usage so much less frustrating. Nice work @ladeirarodolfo!

**Pull request merged**